keras_hub/src/layers/modeling/reversible_embedding.py

```diff
@@ -115,7 +115,12 @@ def build(self, inputs_shape=None):
     def call(self, inputs, reverse=False):
         if reverse:
             if self.tie_weights:
-                kernel = ops.transpose(ops.convert_to_tensor(self.embeddings))
+                # Ensure embeddings is properly converted to a tensor
+                embeddings_tensor = self.embeddings
+                # If it's a Keras variable, get its value
+                if hasattr(embeddings_tensor, "value"):
```

Comment:
so what's the error here? embeddings_tensor has no value when tf numpy is enabled? what's the error?